FactoryMethod/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.NetMentoring.FactoryMethod
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            List<List<FeedItem>> listOfFeeds = new List<List<FeedItem>>();
```

Comment:
Please at least one real test - now this code does nothing(list is empty)

Reply:
ok